spec/support/api/helpers.rb

```diff
@@ -70,6 +70,21 @@ def subcollection_action_identifier(type, subtype, action, method = :post)
           end
         end
 
+        def subcollection_action_classed_identifiers(type, subtype, action, method = :post, klass)
+          subtype_actions = "#{subtype}_subcollection_actions".to_sym
+          if ::Api::ApiConfig.collections[type][subtype_actions]
+            ::Api::ApiConfig
+              .collections[type][subtype_actions][method]
+              .detect { |spec| spec[:name] == action.to_s }[:identifiers]
+              .detect { |spec| spec[:klass] == klass.to_s }[:identifier]
+          else
```

Comment:
Okay, fine, I actually have no idea what is supposed to happen if there aren't subtype actions present for multiple ids, my initial idea is that we probably don't need the else at all but who even knows.

Reply:
you do need the else 👍  subcollection actions/identifier can be declared on the collection itself (i.e. a subcollection action's identifiers are defined per collection), or generically as part of the subcollection itself (the else case).

btw, it there  way to generalize the following pattern in a method, I see it twice here plus in the collection_action_classed_identifier.

```
              .collections[type][action_type][method]
              .detect { |spec| spec[:name] == action.to_s }[:identifiers]
              .detect { |spec| spec[:klass] == klass.to_s }[:identifier]
```

Reply:
Yeah, I was gonna get to that in refactoring, I needed to close the bug first though.